Add default settings to example in README.md

tassoevan opened the pull request:
Added for a quick reference of settings that can be used in `Slim\Logger\DateTimeFileWriter` constructor.

tassoevan committed:
Add default settings to example in README.md

Added for a quick reference of settings that can be used in `Slim\Logger\DateTimeFileWriter` constructor.

silentworks commented:
This is kinda saying to a end user that they should have to set this at all times even if they just want to use the default settings. Create a second example below the original showing the values that can be changed with values different to the default ones, then I will merge the PR.

tassoevan commented:
I agree. :+1:

tassoevan committed:
Fix README.md examples

Fixed according with PHPDoc for method `\Slim\Logger\DateTimeFileWriter::__construct()`.
